posthog-react-native/src/posthog-rn.ts

```diff
@@ -66,9 +70,7 @@ export class PostHog extends PostHogCore {
 
     // Ensure the async storage has been preloaded (this call is cached)
     const setupAsync = async (): Promise<void> => {
-      if (!this._asyncStorage?.isPreloaded) {
-        await (this._asyncStorage as SemiAsyncStorage)?.preloadAsync()
-      }
+      await (this._asyncStorage as SemiAsyncStorage)?.preloadAsync()
```

Comment:
`preloadAsync` will check `isPreloaded` already

Reply:
Got rid of it alltogether